meta-flutter/recipes-flutter/embedded-linux/flutter-client_git.bb

```diff
@@ -0,0 +1,19 @@
+SUMMARY = "Wayland backend (Stand-alone Wayland app)"
```

Comment:
File name should be `flutter-wayland-client`

Reply:
I fixed it.

Reply:
You didn't do that.

Reply:
Sorry, I forgot to fix it. I fixed it.